### PR TITLE
geo: Switch to geoip2 mmdb format

### PR DESCRIPTION
**Pull request by mattrobenolt**
cc @billyvg

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1421,9 +1421,6 @@ def get_sentry_sdk_config():
 
 # GeoIP
 # Used for looking up IP addresses.
-# For example /usr/local/share/GeoIP/GeoIPCity.dat
-GEOIP_PATH = None
-# Same file but in the newer format. Both are required.
 # For example /usr/local/share/GeoIP/GeoIPCity.mmdb
```
Comment (billyvg) [inline src/sentry/conf/server.py]:
```suggestion
# For example /usr/local/share/GeoIP/GeoIPCity.mmdb
# See: https://dev.maxmind.com/geoip/geoip2/geolite2/
# or direct download here:
# https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz
```

File: src/sentry/utils/geo.py [hunk 2]
```diff
@@ -19,33 +21,40 @@ def geo_by_addr(ip):
 def _init_geoip():
     global geo_by_addr
     try:
-        import GeoIP
+        import maxminddb
```
Comment (billyvg) [inline src/sentry/utils/geo.py]: Hmm should we be using `maxminddb` on reload (it's currently using `geoip2`)

**Comment by NullYing** [comment]
The Legacy "dat" format DBs are discontinued:

https://support.maxmind.com/geolite-legacy-discontinuation-notice/

**Comment by mattrobenolt** [comment]
Yeah, hence this change.

**Comment by NullYing** [comment]
9.1的修复版能合上这个mr吗？没有data文件这个功能几乎无法使用。

> Yeah, hence this change.

**Comment by EvanPurkhiser** [comment]
What's blocking on getting this out?

**Comment by mattrobenolt** [comment]
I dunno, I think it's fine. We can just merge it and see what happens.

**Comment by mattrobenolt** [comment]
I was mostly afraid of change in data being returned, but I don't think it's enough of a concern. I've already been using this patch elsewhere, so it does work.

**Comment by NullYing** [comment]
Great, let's try？

**Comment by mattrobenolt** [comment]
@NullYing fwiw, this has been cherry-picked into 9.1.1, so it shouldn't really affect you. The latest release has this patch.

**Review by evanpurkhiser — APPROVED** [review]

**Comment by NullYing** [comment]
image

Where will it take effect?
Looks like it don't work

**Comment by mattrobenolt** [comment]
I'm not sure, but this is a better question at this point for our forums. I know nothing about how you're configuring this or what you're doing. Or even if you're running 9.1.1.

https://forum.sentry.io/

**Comment by NullYing** [comment]
image

I try in sentry shell, it is work

**Comment by mattrobenolt** [comment]
forum.sentry.io

**Pull request merged**